src/MoBi.Presentation/Tasks/Interaction/InteractionTaskContext.cs

```diff
@@ -47,6 +49,16 @@ public interface IInteractionTaskContext
       string GetTypeFor<T>();
 
       void UpdateTemplateDirectories();
+
+      string RenameFor(IObjectBase objectToRename, IReadOnlyList<string> forbiddenNames);
```

Comment:
why does it look exactly like the interface? Not sure

Reply:
I don't understand what this means

Reply:
it is exactly the same definiton as the interface you created above. Why duplicate? RenameFor and NewName